www/src/styles/container.css

```
@@ -0,0 +1,46 @@
+  }
+}
+
+.content {
+  min-height: 400px;
+  padding: 0 20px 0 320px;
+
+  h1,
+  .page-title {
+    padding-top: 30px;
+    margin-bottom: 30px;
+    margin: 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)